doc/source/data-ingest.rst

```diff
@@ -0,0 +1,140 @@
+.. _data-ingest:
+
+.. toctree::
+    :hidden:
+
+    conduce-entities
+
+====================
+Intro to Data Ingest
+====================
+
+Visualizing your data in Conduce begins with ingest.  Ingest is the process translating your data into a format that Conduce understands.  Conduce uses entities to define unique objects.  Entities are ingested into datasets.  Datasets hold collections of related entities and provide the interface a user needs to connect their data to a visualization renderer.
+
+Conduce needs a few key pieces of information in order to visualize your data in space and time, those are:
+
+**Identity**
+   Identities distinguish one entity from another.  An entity exists in Conduce as one more more samples.  All samples that describe the same entity share an identity.
+**Location**
+   An entities location describes where in space the entity exists.  Locations are defined in units according to the coordinate system in which they are displayed. (Latitude and Longitude on a world map.)
+**Time**
+   Time defines the moment in time at which the rest of the information describing an entity is valid.
```

Comment:
Several comments on time

1. You should remind the reader that time is only required for samples and is implicitly infinite for entities
2. You should point out that a sample is uniquely identified by its identity and time.  That is, only one sample may be specified with a given identity and time
3. Since time is more fundamental than location, it might be worth putting it second after identity

Reply:
originally +1 to all, esp. 2 & 3

Though I should also point out, time is only included for sample data, not for static data

Reply:
I'd reword "entity is valid" --> "entity was captured"

Reply:
I agree with your points. I think that I make them at various points (redundantly even) throughout the documentation.  For definitions, the key is to find wording that is brief and understandable yet meaningful and complete.

I debated whether to include this definitions section in this document at all.  Now I'm lean toward no.

Reply:
I think "captured" assumes too much.  I don't know when (or if) the data was captured and that term does not communicate how conduce interprets the data.